0-ad-history-mod-package/simulation/templates/structures/athen/gymnasium.xml

```diff
@@ -28,6 +28,10 @@
     <Requirements>
       <Techs datatype="tokens">-phase_city phase_town</Techs>
     </Requirements>
+    <History>
+        The gymnasium (derived from “gymnos”, a Greek term for nudity) was a facility for physical exercise. It essentially consisted of a large field functioning as an exercise area, surrounded by changing and practicing rooms. Gymnasia were constructed all over Greece, some smaller, some larger, sometimes even with spectator stands like stadiums. Many young men (it was exclusive to male citizens) spent large portions of their day at the gymnasium practicing all kinds of sports like jumping, wrestling, boxing, javelin and discus throwing. As the name suggests, in the gymnasium, it was customary for athletes to exercise completely naked. Contests in these various sports were also held, often on the occasion of religious rituals and festivals. The most infamous one, of course, being the Olympic Games in Olympia.
+        Because the Ancient Greeks believed in a strong connection between education and sports, the gymnasium over time also became a place of intellectual endeavor where lectures and speeches were taken and an important center for philosophers. Specific gymnasia were often associated with specific schools of philosophy.
```

Comment:
```suggestion
        Because the Ancient Greeks believed in a strong connection between education and sports, the gymnasium over time also became a place of intellectual endeavor where lectures and speeches were given. Specific gymnasia were often associated with specific schools of philosophy.